### ISSUE TITLE
Update from 0.3.17.29 to 0.3.21 failed

### ISSUE THREAD

**Issue by bigretromike** [anime-db/anime-db#161]
I used official download to get exe (for windows 10)
I pick update 
it found something and crashed and now progress is in loop and there was no indicator of fail
only the execution log with this:

```
Search for a new version of the application
Discovered a new version of the application: v0.3.21
  - Installing anime-db/anime-db (v0.3.21)
    Downloading

Update itself has been completed
> AnimeDb\Bundle\AnimeDbBundle\Composer\ScriptHandler::backupDB
> AnimeDb\Bundle\AnimeDbBundle\Composer\ScriptHandler::installConfig
Loading composer repositories with package information
Updating dependencies
                                                                            

  [Symfony\Component\Debug\Exception\OutOfMemoryException]                     
  Error: Allowed memory size of 134217728 bytes exhausted (tried to allocate   
  1519603 bytes)                                                               

animedb:update

```

As you mention in #158 

```
bin\php\php.exe composer.phar self-update
bin\php\php.exe composer.phar update
```

Did the trick but handle it inside webui would be also a good thing, or tweak memory limit in php configuration that is bundled with win version.

**Comment by bigretromike** [comment]
But that is still only workaround, any change to make it more stable ?

**Comment by peter-gribanov** [comment]
@bigretromike try run the following commands in Windows command line (cmd):
```
echo. >> bin\php\php.ini && echo memory_limit=1G >> bin\php\php.ini && bin\php\php.exe composer.phar update && bin\php\php.exe app\console animedb:update
```

**Comment by bigretromike** [comment]
Sure, it helped but its not out of box :-)

On July 31, 2016 10:10:20 PM GMT+02:00, Peter Gribanov <notifications@github.com> wrote:
>@bigretromike try run the following commands in Windows command line
>(cmd):
>```
>echo. >> bin\php\php.ini && echo memory_limit=1G >> bin\php\php.ini &&
>bin\php\php.exe composer.phar update && bin\php\php.exe app\console
>animedb:update
>```
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/anime-db/anime-db/issues/161#issuecomment-236453687

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by peter-gribanov** [comment]
@bigretromike the problem exists only in older versions of the application. The problem is fixed in version 0.3.18

**Comment by bigretromike** [comment]
Thank you

On August 1, 2016 1:20:54 PM GMT+02:00, Peter Gribanov <notifications@github.com> wrote:
>@bigretromike the problem exists only in older versions of the
>application. The problem is fixed in version 0.3.18
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/anime-db/anime-db/issues/161#issuecomment-236554749

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
